scr/controllers/controllers.js

```diff
@@ -0,0 +1,183 @@
+const controller = {};
+const {connector} = require('../../config/mysql_db')
+const  multer = require('multer')
+const storage = require('../../config/multer');
+const uploader = multer({storage});
+
+
+controller.list = (req, res) => {
+        let sql = 'SELECT * FROM new_product';
```

Comment:
the connection to the database must be in another layer. Please do some research about the Data Access layer or how to separate the data base access from the controller

Reply:
could you be more specific please? I made the db access (data and connection function) in another js module (config/mysql_db). In this case in controllers i'm just calling the db with an import

Reply:
Ok, don't worry about this now, when all the changes have been applied I can check these cases again

Reply:
Ok, I'll leave it like this for now. I have the others changes ready, you can take a look at the changes